components/identity-core/org.wso2.carbon.identity.core/src/test/java/org/wso2/carbon/identity/core/internal/DefaultServiceURLBuilderTest.java

```diff
@@ -252,44 +255,47 @@ public void testBuild() {
         }
 
         return new Object[][]{
-                {"https", "www.wso2.com", 9443, null, "", fragmentParams,
-                        "https://www.wso2.com:9443#key1%3Dfragment%26key2%3Dfragment%26key3%3Dfragment%26key4%3Dfragment",
-                        ""},
-                {"https", "www.wso2.com", 9443, null, "fragment", fragmentParams,
-                        "https://www.wso2.com:9443/samlsso#fragment", "/samlsso"},
-                {"https", "www.wso2.com", 9443, null, "", fragmentParams,
-                        "https://www.wso2.com:9443/samlsso#key1%3Dfragment%26key2%3Dfragment%26key3%3Dfragment%26key4%3Dfragment",
+                {"https", "www.wso2.com", 9443, "/proxyContext", null, "", fragmentParams,
```

Comment:
Since this is used multiple times lets move it to a constant

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/3063.